### PR TITLE
#160950209 Design Landing page

### PR DESCRIPTION
**Pull request by iamuchejude**
#### What does this PR do?
Create a landing(homepage) page for the app 

#### Description of Task to be completed?
When a user visits the platform, they should be able to see the Landing page. For now, you can set up the page with dummy data as soon as the 'Create Articles' feature is completed, it will be updated to work with dynamic data  

#### How should this be manually tested?
* Clone the repository
* CD into `thor-ah-frontend`
* Run `npm install`
* Run `npm start`
* Navigate to the `index` page

#### What are the relevant pivotal tracker stories?
[#160950209](https://www.pivotaltracker.com/story/show/160950209)

### Screenshots (if appropriate)
localhost_3000_ 1
localhost_3000_ 2
localhost_3000_ 3
localhost_3002_ 1
localhost_3002_

### CODE REVIEW

File: src/styles/App.css [hunk 1]
```diff
@@ -1,40 +1,21 @@
-.App {
-  text-align: center;
+* {
+  box-sizing: border-box;
+  outline: none;
+  font-family: 'Nunito Sans', sans-serif;
```
Comment (lauragift21) [inline src/styles/App.css]: The default font family for the application should be Lato. So you can update this.

File: public/index.html [hunk 2]
```diff
@@ -5,36 +5,14 @@
...
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>React App</title>
+    <link href="https://fonts.googleapis.com/css?family=Nunito+Sans:400,700" rel="stylesheet">
```
Comment (lauragift21) [inline public/index.html:10]: Same thing as the change requested in the App.css file. The default font family should be `font-family: Lato`, So you can update this too.

**Review by lauragift21 — CHANGES_REQUESTED** [review]
LGTM 👍. I added a few comments for you check and update.

**Review by emekafredy — CHANGES_REQUESTED** [review]
Good work! The Article Categories can be located in the mobile version, but not the Desktop version. Please correct this.

File: src/components/Articles/Articles.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  }
+
+  componentWillMount() {
+    this.setState({
+      articles: [
+        {
+          id: 1,
+          title: 'This Article talks about some Tall Buildings And City Night Life',
+          slug: 'this-is-slug',
+          body: 'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
```
Comment (johnkingzy) [inline src/components/Articles/Articles.js:18]: Is there any reason why you are setting the state in componentWillMount? I will suggest you set this directly in the initial state i.e `this.state = { articles: [ ......... ] }`

File: src/components/Articles/Articles.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+  render() {
+    const { articles } = this.state;
+    const { content } = this.props;
+    return (
+      <div className={ styles.articles }>
+        <div data-content="featured" className={ content === 'featured' ? styles.active : '' }>
+          { articles.map(article => {
+            const { id, title, slug, thumbnail, body } = article;
+            const snippet = body;
+            const details = {
```
Comment (johnkingzy) [inline src/components/Articles/Articles.js:62]: I will suggest you add a condition to display a message if no article is found

File: src/components/Category/Category.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import styles from './category.module.scss';
```
Comment (johnkingzy) [inline src/components/Category/Category.js:5]: the previous comment also applies. Please add some comment between these lines

File: src/components/Header/Header.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import styles from './header.module.scss'
+import Search from '../Search/Search';
+import Category from '../Category/Category';
```
Comment (johnkingzy) [inline src/components/Header/Header.js:6]: Previous comment also applies, please add comments between the imports

File: src/components/Header/Menu.js [hunk 7]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { FaBell } from 'react-icons/fa';
+import styles from './header.module.scss';
+
+const  photoStyle = {
+  height: '40px',
+  width: '40px',
+  border: '1px solid #000',
```
Comment (johnkingzy) [inline src/components/Header/Menu.js]: I will suggest you move this style into a different file

**Review by johnkingzy — CHANGES_REQUESTED** [review]
Nice one. Good job so far

**Review by lauragift21 — APPROVED** [review]

**Review by emekafredy — APPROVED** [review]
Looks Good

**Review by johnkingzy — APPROVED** [review]

**Pull request merged**